dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/imports/events/EventsTests.java

```diff
@@ -136,6 +137,57 @@ void eventsImportNewEventsFromFile(String fileName, String contentType) throws E
     response.validate().statusCode(200).body("status", equalTo("OK"));
   }
 
+  @Test
+  void eventsImportNewEventsWithInvalidUidForJson() throws Exception {
+    Object obj =
+        new FileReaderUtils()
+            .read(new File("src/test/resources/tracker/importer/events/event.json"))
+            .replacePropertyValuesWith("event", "invalid_uid")
+            .get();
+
+    ApiResponse response =
+        trackerImportExportActions.post(
+            "",
+            ContentType.JSON.toString(),
+            obj,
+            new QueryParamsBuilder()
+                .addAll("dryRun=false", "eventIdScheme=UID", "orgUnitIdScheme=UID"));
```

Comment:
IdSchemes are only supported for metadata. `eventIdScheme` does not exist. Is it even necessary to add these params? UID is the default